src/user/redux/modules/change_password.js

```diff
@@ -1,12 +1,20 @@
-import { FETCHED_MEMBER } from '../../../shared/redux/modules/member.js'
+import { createAction } from 'redux-actions'
```

Comment:
Let's keep flow on every module. Add a `/* @flow */` annotation to the top and see if you can resolve the errors (you'll need to add a couple of type signatures)